DoctorWho/DoctorWho.Db/DoctorWhoCoreDbContext.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace DoctorWho.Db
+{
+    public class DoctorWhoCoreDbContext : DbContext
+    {
+        public static DoctorWhoCoreDbContext _context = new DoctorWhoCoreDbContext();
```

Comment:
we should never make the context as a singleton, here you will have one context that will be accessed by all calls, and will cause concurrency issues. This will be more of an issue when we work on the APIs project. Consider removing this an initializing a new context for every repo in their constructor.